tests/test_vqe.py

```diff
@@ -895,17 +895,21 @@ def ansatz(params, **kwargs):
         mt = qml.metric_tensor(qnodes)(p)
         assert qml.tape_mode_active()  # Check that tape mode is still active
 
-        qml.disable_tape()
+        try:
+            qml.disable_tape()
```

Comment:
why would we want to not just test the new tape-based metric tensor here?

Reply:
Oh you mean delete this portion?